k8s-ingress/README.md

```diff
@@ -0,0 +1,96 @@
+# Linkerd as Kubernetes Ingress Controller
+
+This example application demonstrates the basics of using linkerd as an
+ingress controller. It consists of:
+* `ingress-controller.yaml`: A linkerd config and linkerd deployment listening on `hostPort: 80`.
+* `services.yaml`: Deployment and service definitions for a number of seal-related example services.
+* `ingress.yaml`: An ingress resource that defines routing to those services.
+
+## Deploy
+
+### Ingress Controller
+
+```bash
+$ kubectl apply -f ingress-controller.yaml
+```
+
+Verify linkerd pods:
+
+```bash
+$ kubectl get po --namespace=l5d-system
+NAME                  READY     STATUS    RESTARTS   AGE
+l5d-932856984-v906t   2/2       Running   0          34m
+```
+
+Open admin ui (minikube specific):
+
+```bash
+$ minikube service l5d -n l5d-system --url | tail -n1 | xargs open
+```
```

Comment:
Yeah, I need to do more testing on gke anyhow. But also brings up a good point. Is this example meant for beginners (whom I suspect don't have access to a gce cluster), or is this an example of a fully fledged production ready ingress controller? I think as we get closer to the edge, environments start to diverge more and we won't be able to rely on consistency between k8s setups. Just something to consider.